app/models/manageiq/providers/azure/manager_mixin.rb

```diff
@@ -14,6 +14,20 @@ def verify_credentials(_auth_type = nil, options = {})
     connect(options)
   end
 
+  def edit_with_params(params)
+    default_endpoint = params.delete("endpoints").dig("default")
+    default_authentication = params.delete("authentications").dig("default")
+
+    tap do |ems|
+      ems.default_authentication.assign_attributes(default_authentication)
```

Comment:
Out of curiosity, why `assign_attributes`?... elsewhere in code we use `attributes=`

Reply:
Well, the argument is just a subset of attributes, not all of them. I'm not sure how `attributes=˙ handles the argument if it's just a subset :confused: 

Also I'm using it already in the amazon provider :trollface: